src/core/lib/iomgr/parse_address.cc

```diff
@@ -16,15 +16,15 @@
  *
  */
 
+#include "src/core/lib/iomgr/parse_address.h"
+
 #include <grpc/support/port_platform.h>
```

Comment:
port_platform.h has to be the very first include, for portability reasons.

It looks like the order of the includes was already wrong in this file, and this change just made it more confusing.  Other than port_platform.h being the first, we should follow the order specified in the style guide:

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

So I think the right order for this file is the following:

```
#include <grpc/support/port_platform.h>

#include "src/core/lib/iomgr/parse_address.h"

#ifdef GRPC_POSIX_SOCKET
#include <errno.h>
#include <net/if.h>
#endif
#include <stdio.h>
#include <string.h>
#ifdef GRPC_HAVE_UNIX_SOCKET
#include <sys/un.h>
#endif

#include <grpc/support/alloc.h>
#include <grpc/support/log.h>
#include <grpc/support/string_util.h>

#include "src/core/lib/gpr/string.h"
#include "src/core/lib/gprpp/host_port.h"
#include "src/core/lib/iomgr/grpc_if_nametoindex.h"
#include "src/core/lib/iomgr/sockaddr.h"
#include "src/core/lib/iomgr/socket_utils.h"
```

Similar comment in all of the other files where the includes were reordered.  If these changes were made by clang-format, then I wonder if you're using a different version of clang-format than we are?  You should be able to get the right result by running `tools/distrib/clang_format_code.sh`.

Reply:
Interesting. I ran `CLANG_FORMAT_SKIP_DOCKER=true tools/distrib/clang_format_code.sh`. I'll put them back.